fake_switches/brocade/command_processor/enabled.py

```diff
@@ -67,7 +67,7 @@ def do_skip_page_display(self, *args):
         pass
 
     def do_write(self, *args):
-        pass
+        self.switch_configuration.commit()
```

Comment:
OK, my comment is invalid.  It's the name `switch_configuration` that is misleading, for me.  I agree with your change. :+1: